core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -923,11 +923,20 @@ public String toString() {
   public static final PropertyKey MASTER_HEARTBEAT_INTERVAL_MS =
       new Builder(Name.MASTER_HEARTBEAT_INTERVAL_MS)
           .setAlias(new String[]{"alluxio.master.heartbeat.interval.ms"})
-          .setDefaultValue("1sec")
+          .setDefaultValue("1min")
```

Comment:
Why is this so long? I would think something like 5 or 10 seconds should work?

Reply:
Change to 10 seconds. We only need the heartbeat for config checker, so we may not need the frequent heartbeat.